volatility3/framework/layers/intel.py

```diff
@@ -203,10 +209,10 @@ def _translate_entry(self, offset: int) -> Tuple[int, int]:
                     "Page Fault at entry " + hex(entry) + " in table " + name,
                 )
             # Check if we're a large page
-            if large_page and (entry & (1 << 7)):
+            if large_page and (entry & (1 << linux_constants.PAGE_BIT_PSE)):
                 # Mask off the PAT bit
-                if entry & (1 << 12):
-                    entry -= 1 << 12
+                if entry & (1 << linux_constants.PAGE_BIT_PAT_LARGE):
```

Comment:
Right. Where do you think it's the best place to move them? As a class attribute or as a global constant at the top of the intel.py file?

Reply:
Hmmm, I feel they probably should live in the class?  I guess that'd only be an additive change, so just a bump of the MINOR version on the framework.  It would be the first referable constant we don't keep in the `constants` files though?  I'm torn, which do you prefer?

Reply:
Yeah, I suppose only the layers will use them, but I don't think it's worth try to hide them if it means making the code ugly

Reply:
I'll make them internal attributes. I would prefer to keep them uppercase to emphasize that they are constants, but if you want me to follow the lowercase style please let me know.

```python
class Intel(linear.LinearlyMappedLayer):
    _PAGE_BIT_PRESENT = 0
    _PAGE_PRESENT = 1 << _PAGE_BIT_PRESENT
...
    _entry_format = "<I"
    _page_size_in_bits = 12
    _bits_per_register = 32
```

Reply:
Ok, done.. please have a look